### PR TITLE
owners don't consume a license

### PR DESCRIPTION
**Pull request by brntbeer**
### Why:

Owners don't consume a license unless they're in an organization

### What's being changed:

calling it out here. unsure where else it could be mentioned  but this is a great start

### Check off the following:
- [ ] I have reviewed my changes in staging. (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**)
- [ ] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [ ] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

**Comment by brntbeer** [comment]
> * [ ]  For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
> *

I could definitely benefit from some help here. The way i've phrased the line is not that clear i think even for me =)

**Comment by brntbeer** [comment]
Additionally, we could do like is done in https://github.com/github/docs/blob/main/content/github/setting-up-and-managing-organizations-and-teams/adding-a-billing-manager-to-your-organization.md, and add the Note for billing managers and enterprise owners.

**Comment by janiceilene** [comment]
Thanks for opening a PR @brntbeer 💖 I'll get this triaged for review ⚡

**Comment by brntbeer** [comment]
bump. @janiceilene any luck getting review?

**Comment by janiceilene** [comment]
I'll check with the team @brntbeer 💛 Thanks for the bump!

File: content/github/setting-up-and-managing-your-enterprise/roles-in-an-enterprise.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ Enterprise owners have complete control over the enterprise and can take every a
 
 Enterprise owners cannot access organization settings or content unless they are made an organization owner or given direct access to an organization-owned repository. Similarly, owners of organizations in your enterprise do not have access to the enterprise itself unless you make them enterprise owners.
 
-You can add as many enterprise owners as you'd like to your enterprise. {% if currentVersion == "free-pro-team@latest" %}Enterprise owners must have a personal account on {% data variables.product.prodname_dotcom %}.{% endif %} As a best practice, we recommend making only a few people in your company enterprise owners, to reduce the risk to your business.
+You can add as many enterprise owners as you'd like to your enterprise as they do not consume a license unless they are an owner or member of at least one organization within the enterprise. {% if currentVersion == "free-pro-team@latest" %}Enterprise owners must have a personal account on {% data variables.product.prodname_dotcom %}.{% endif %} As a best practice, we recommend making only a few people in your company enterprise owners, to reduce the risk to your business.
```
Comment (megbird) [inline content/github/setting-up-and-managing-your-enterprise/roles-in-an-enterprise.md:34]: I actually think your rewording is fine 😄  If you wanted it to be a bit more concise you can also do this:

```suggestion
An enterprise owner will only consume a license if they are an owner or member of at least one organization within the enterprise. {% if currentVersion == "free-pro-team@latest" %}Enterprise owners must have a personal account on {% data variables.product.prodname_dotcom %}.{% endif %} As a best practice, we recommend making only a few people in your company enterprise owners, to reduce the risk to your business.
```

There's a section for billing managers down below as well that we can update with similar wording (but I'm guessing this is also true for billing managers, yeah?)

**Review by megbird — APPROVED** [review]
@brntbeer thanks for the gentle bump on this! I had a few suggestions that I think are valid but let me know 😄

File: content/github/setting-up-and-managing-your-enterprise/roles-in-an-enterprise.md (hunk 1 above)
Comment (brntbeer) [reply content/github/setting-up-and-managing-your-enterprise/roles-in-an-enterprise.md:34]: Will adopt the same wording for billing managers! Thanks @megbird

**Review by megbird — APPROVED** [review]
Amazing! ⚡ merging now

**Pull request merged**